src/resources/postcss/customizer-controls.pcss

```diff
@@ -0,0 +1,304 @@
+/* = Customizer Controls (V2)
+============================ */
+
+.wp-customizer {
+
+	[id^="customize-control-tribe_customizer"] {
+
+		.customize-control-description {
+			font-style: normal;
+			margin-bottom: 12px;
+		}
+
+		.customize-control-title {
+			margin-bottom: 0;
+
+			& ~ .customize-control-content {
+				margin-top: 8px;
+			}
+
+			& ~ .customize-control-description {
+				margin-top: 4px;
+				margin-bottom: 3px;
+			}
+		}
+
+		.customize-control .customize-inside-control-row {
+			padding: 3px 0;
+		}
+
+		.button.wp-color-result {
+			border-color: #141827;
+
+			&:active,
+			&:focus,
+			&:hover {
+				border-color: #141827;
+				box-shadow: 0px 0px 0px .75px rgba(20, 24, 39, .5);
+			}
+		}
+
+		/* This handles a hidden color control based on a radio option - brings it closer to the radio control. */
+		/* Note it requires the radio name end in "_choice" and the color control end in "_color" */
+		&[id$="_choice"].customize-control-radio + [id^="customize-control-tribe_customizer"][id$="_color"] {
+			margin-top: -18px;
+		}
+
+
+		/* Range Slider, TEC-style */
+
+		.tec-range-slider-label {
+			border: 0;
+			clip: rect(0 0 0 0);
+			height: 1px;
+			margin: -1px;
+			overflow: hidden;
+			padding: 0;
+			position: absolute;
+			width: 1px;
+		}
+
+		input[type=range].tec-range-slider {
+			background-color: transparent;
+			border-radius: 9px;
+			width: calc(100% - 24px);
+
+			&:active,
+			&:focus,
+			&:focus-visible {
+				box-shadow: 0 0 0 2px #334AFF;
+				outline: none;
+			}
+
+			/* Track styles for allthebrowsers */
+
+			&::-moz-range-track {
+				background-color:  #FFF;
+				border-radius: 9px;
+				border: 0;
+				cursor: pointer;
+				height: 18px;
+				width: 100%;
+			}
+
+			&::-webkit-slider-runnable-track {
+				background-color:  #334AFF;
+				border-radius: 9px;
+				border: 0;
+				cursor: pointer;
+				height: 18px;
+				width: 100%;
+			}
+
+			&::-ms-track {
+				background-color:  #FFF;
+				border-radius: 9px;
+				border: 0;
+				cursor: pointer;
+				height: 18px;
+				width: 100%;
+			}
+
+			&::-moz-range-progress {
+				background-color:  #334AFF;
+				border-radius: 9px;
+				height: 16px;
+			}
+
+			&::-webkit-progress-value {
+				background-color:  #334AFF;
+			}
+
+			&::-ms-fill-lower {
+				background-color:  #334AFF;
+			}
+
+			/* Thumb styles for allthebrowsers */
+
+			&::-moz-range-thumb {
+				background: #ffffff;
+				border-radius: 9px;
+				border: 1px solid #141827;
+				cursor: pointer;
```

Comment:
if we import variables, this won't be a problem, but we have a mix of uppercase and lowercase for colors.